### ISSUE TITLE
Trainer not properly validating against train data.

### ISSUE THREAD

**Issue by geekmdtravis** [geekmdtravis/cs-7643-final-project#61]
The trainer is simply training twice, once on train and again on validation, then stepping against validation loss. Thus, the losses were tracking nearly identically.
